examples/fabric/cloud_router/example_1/csharp/equinix-fabric-cloud_router-example_1.csproj

```diff
@@ -7,7 +7,7 @@
 	</PropertyGroup>
 	<ItemGroup>
 		<PackageReference Include="Pulumi" Version="3.*" />
-		<PackageReference Include="Pulumi.Equinix" Version="(, 1.0.0)" />
+		<PackageReference Include="Pulumi.Equinix" Version="0.18.1-alpha.1729554944+37549499" />
```

Comment:
@ocobles how do we keep this version from changing?  IIRC the `(, 1.0.0)` was a fix you put in to address the mismatch between example versions and release versions in the docs and we want to ensure it stays in place.

Reply:
Ah, good point. I was curious why these files were changing when doing the `make build` after just adding mappings and nothing else.

Would love to find out how to retain this.

Reply:
It looks like the generator script was not always figuring out the correct version number for replacement.  I opened #227 which appears to fix the issue.